src/bin/psql/describe.c

```diff
@@ -2478,12 +2482,24 @@ describeOneTableDetails(const char *schemaname,
 							  schemaname, relationname);
 			break;
 		case RELKIND_MATVIEW:
-			if (tableinfo.relpersistence == 'u')
-				printfPQExpBuffer(&title, _("Unlogged materialized view \"%s.%s\""),
-								  schemaname, relationname);
+			if (!tableinfo.isdynamic)
+			{
+				if (tableinfo.relpersistence == 'u')
```

Comment:
whats a use case for this?

```
reshke=# create unlogged materialized view mv1 as select;
ERROR:  materialized views cannot be unlogged
```

Reply:
Good catch.
I just follow the places with materialized views, PG has disallowed unlogged matview, but the codes left.

Reply:
It may worth a patch to Postgres:)